scripts/post-rewrite

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+rewriteType=$1
+
+if [ "$rewriteType" != "amend" ]; then
+  exit 0
+fi
+
+function getCoauthor {
+  signiture=$(git log --no-merges -1 --author "$1" --format='%an <%ae>' 2>/dev/null)
+  echo "Co-authored-by: $signiture"
+}
+
+commitMessage=$(git log --format=%B -n 1 HEAD)
+coworkers=$(git config --get-all coworking.coauthor)
+
+commitMessage="$commitMessage\n"
+while read -r coworker; do
+  signature=$(getCoauthor $coworker)
+  if [[ $commitMessage != *"$signature"* ]]; then
+    commitMessage="$commitMessage\n$signature"
+  fi
+done <<< "$coworkers"
+
+git commit -m "$(echo -e "$commitMessage")" --amend --no-verify
```

Comment:
TODO: Mabe this could just trigger the `commit-msg` hook since it has all the functionality.